packages/notification-center/src/shared/config/notificationItemButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { ButtonTypeEnum } from '@novu/shared';
+
+export interface INotificationButtonStyles {
+  key: ButtonTypeEnum;
+  value: IButtonStyles;
+}
+
+export interface IButtonStyles {
+  color: string;
+}
+
+const primaryButton: INotificationButtonStyles = {
+  key: ButtonTypeEnum.PRIMARY,
+  value: {
+    color: 'blue',
```

Comment:
What are those? Do we actually do them blue?

Reply:
Yea those are the reused design for the buttons